admin/spec/features/properties.rb

```diff
@@ -6,19 +6,19 @@
   before { sign_in create(:admin_user, email: 'admin@example.com') }
 
   it "lists properties and allows deleting them" do
-    create(:property, name: "Type", presentation: "Type")
+    create(:property, name: "Type prop", presentation: "Type prop")
```

Comment:
Changed "Type" to "Type prop" because of text Option types" used for the menu item which test recognizes as non deleted.